README.md

```diff
@@ -16,8 +16,12 @@
 Add your answers to this markdown file.
 
 
-1. One reason asymptotic analysis may be misleading is when using smaller inputs such as a small array that needs to be sorted. When doing asysmptotic analysis, we ignore constant factors, but these constant factors can make a big difference to the run time if we are dealing with a smaller array. Another reason asymptotic analysis may be misleading is that it can assume that we do not have any knowledge of the data. For example, we may collect data that is already sorted or close to sorted. So if we want to choose a sorting algorithm to sort that data, we may look at the average case run times for all sorting algorithms and choose the fastest one. But if we know that the data is already sorted or close to sorted, we could choose an algorithm like insertion sort which has one of the worst average case run times but one of the best best case run times. Finally, a third reason why asymptotic analysis can be misleading is that it ignores hardware factors. The run time for certain algorithms may be faster, but require more memory from the device. This could cause problems if you want to work with a faster algorithm on a device that does not have enough memory to support that algorithm.
+1. One reason asymptotic analysis may be misleading is that we ignore constant factors within the algorithms, but constant factors can have a large effect on the overall runtime. For example, a function could have a linear time complexity, but a function that has runtime 1(n) vs a function that has a lot more constant factors like 1000(n) could be a significant difference in runtime, yet bot functions have the same time complexity. Another reason asymptotic analysis may be misleading is because the bounds are very loose, so the overall time complexity is not very accurate in runtime prediction. Finally, a third reason why asymptotic analysis can be misleading is that it ignores hardware factors. Inspired by the boomboomyoyo repository, asymptotic analysis looks at looks at the input size and factors within an algorithm to predict it's runtime. However, the the computer may be dedicating most or all of it's resources to other programs that are running. This could cause a significant slow down of the runtime of an algorithm regardless of it's time complexity.  
```

Comment:
Wouldn't hardware only account for a constant factor, making it the same as your first reason?